src/main/java/org/folio/inventory/resources/Instances.java

```diff
@@ -66,10 +72,14 @@
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
   private final Storage storage;
   private final HttpClient client;
+  private final InventoryConfiguration config;
+  private static final String BLOCKED_FIELDS_UPDATE_ERROR_MESSAGE = "Unprocessable entity: given Instance is controlled by MARC record, "
+    + "it's blocked fields can not be updated: ";
```

Comment:
I think it should be "These blocked fields can not be updated: " or "it's blocked fields, they can not be updated: "